third_party/silabs/efr32_sdk.gni

```diff
@@ -31,6 +33,8 @@ declare_args() {
 
   kvs_max_entries = 75
 
+  chip_enable_ble_rs911x = false
```

Comment:
move this to the declare_args in `silabs_board.gni` as it was done for other board/hardware specific argument